src/theory/bags/inference_generator.cpp

```diff
@@ -424,31 +424,16 @@ std::tuple<InferInfo, Node, Node> InferenceGenerator::mapDown(Node n, Node e)
   Node sum_zero = d_nm->mkNode(Kind::APPLY_UF, sum, d_zero);
   Node baseCase = d_nm->mkNode(Kind::EQUAL, sum_zero, d_zero);
 
-  // guess the size of the preimage of e
-  Node preImageSize =
-      d_sm->mkSkolemFunction(SkolemFunId::BAGS_MAP_PREIMAGE_SIZE, {f, A, e});
+  // guess the size of distinct elements in A
+  Node size =
+      d_sm->mkSkolemFunction(SkolemFunId::BAGS_MAP_PREIMAGE_SIZE, {f, A});
```

Comment:
Can you update the documentation for the skolem for this? Also the ALF type rule will need to change (https://github.com/cvc5/cvc5/blob/main/proofs/alf/cvc5/theories/Bags.smt3#L36) to remove "U".

Reply:
Done.